### PR TITLE
updating copy + adds btn for @katiebarrett, fixes less compiling

### PR DESCRIPTION
**Pull request by mekarpeles**

### CODE REVIEW

**Comment by mekarpeles** [comment]
cc @katiebarrett

File: openlibrary/templates/site/body.html [hunk 1]
```diff
@@ -1,14 +1,14 @@
 $def with (page)
 
-$ announcement = '<strong>Want to support Open Library?</strong> Until April 30, we\'ll <strong>double your donation</strong>! <a href="https://archive.org/donate?platform=ol">Help us build the great digital library</a>.' 
+$ announcement = 'Is Open Library useful to you? Chip in to keep it growing. Your gift will be matched today! &nbsp; <a href="https://archive.org/donate?platform=ol" class="btn primary">Donate</a>'
```
Comment (bfalling) [inline openlibrary/templates/site/body.html]: QUESTION: Think we should add click tracking?

File: static/css/master.css [hunk 2]
```diff
@@ -5403,7 +5403,12 @@ body#user #description p {font-size:.875em;margin:0 0 30px 0;}
 #lists .listEntry .meta {font-size:.6875em;color:#666;line-height:1.5em;margin:3px 0;}
 
 /* SEARCH INSIDE */
-.snipHL {opacity: 0.20;filter: alpha(opacity = 20);background-color: #00f;position:absolute;}
+.snipHL {
+    opacity: 0.20;
+    filter: alpha(opacity=20);
+    background-color: #00f;
+    position:absolute;
```
Comment (bfalling) [inline static/css/master.css]: QUESTION: Just checking: We still need this `position:absolute`?

File: static/css/master.css [hunk 3]
```diff
@@ -1696,7 +1696,7 @@ fieldset.home legend {
 * html .ui-helper-clearfix { height:1%; }
 .ui-helper-clearfix { display:block; }
 
-.ui-helper-zfix { width: 100%; height: 100%; top: 0; left: 0; position: absolute; opacity: 0; filter:Alpha(Opacity=0); }
+.ui-helper-zfix { width: 100%; height: 100%; top: 0; left: 0; position: absolute; opacity: 0; filter: alpha(opacity=0); }
```
Comment (bfalling) [inline static/css/master.css]: QUESTION: I guess we still need an absolutely positioned item?

Comment (mekarpeles) [reply static/css/master.css]: I think this is for a jquery dialogue box

Comment (mekarpeles) [reply static/css/master.css]: I don't think this selector is used anywhere, rm'ing

**Pull request merged**